Add wallet seed backup notification modal

Sirmorrison opened the pull request:
This PR adds wallet seed backup notification when pages loads for the first time and there is a wallet that needs its seed words to be backed up. 

![image](https://user-images.githubusercontent.com/27733432/136553994-1828bc44-9f81-4aa0-89ec-a099ad416c91.png)

![image](https://user-images.githubusercontent.com/27733432/136554050-9595ad17-a005-402a-a689-6121d636bc18.png)

Sirmorrison committed:
Add wallet seed backup notification modal

dreacot commented on ui/modal/info_modal_layouts.go:
```diff
@@ -101,3 +102,23 @@ func transactionDetailsInfo(th *decredmaterial.Theme) []layout.Widget {
 }
+
+func backupInfo(th *decredmaterial.Theme) []layout.Widget {
+	textGray := "Please backup your seed words and keep them in a safe place in order to recover your funds " +
```
why does this string need to be concatenated?

dreacot reviewed (changes requested):
Currently, each time the user selects backup later and returns to the overview page, the modal pops back up, the user experience for that isn't good, a boolean should be created to check if backup later has be selected and set to true, so the dialog doesn't display for the current user session, until the app is restarted

Sirmorrison committed:
prevent modal from when overview page is loaded
- fix set create password modal notification text size
- fix backup notification modal text size
- clean up code

Sirmorrison committed:
add hover to help and security tools page

dreacot reviewed (approved):
LGTM
